netlify.toml

```diff
@@ -1,12 +1,233 @@
 # Settings in the [build] context are global and are applied to all contexts unless otherwise overridden by more specific contexts.
 [build]
-  command = './netlify/build.sh'
+  publish = "_site/"
+  command = '''
+
+  cp netlify/Gemfile Gemfile && bundle install
+  make bin/helm
+  export PATH=$PATH:$(pwd)/bin
+
+  # LATEST_RELEASE is set in Netlify UI for environment-variables ONLY for docs.projectcalico.org site.
+  # Any branch deployed there will have the production ready build that gets deployed at root /
+
+  if [ -z "$LATEST_RELEASE" ]; then
```

Comment:
Am I correct that we can't use `[context.production]`  because every site in netlify thinks it is building the production site?

Reply:
right, to elaborate on this, since all the sites are `production` we still needed a way to be able to trigger "production" build (which uses base url `/` on `docs.projectcalico.org` netlify site, hence we added an environment variable to the site it self)